airtable/plugin.py

```diff
@@ -30,16 +38,15 @@ class AirtableRecordID:
 
 def airtable_parse_ids_from_url(
     url: str,
-) -> Tuple[AirtableBaseID, Optional[AirtableTableID], Optional[AirtableRecordID]]:
+) -> Tuple[AirtableBaseID, Optional[AirtableTableID], Optional[AirtableRecordID], Optional[AirtableViewID]]:
     """
     Parses Airtable IDs from an Airtable web UI URL.  The minimum required URL format is
     https://airtable.com/{base_id}, with {base_id} being mandatory.  The URL may
     optionally include {table_id}, {view_id}, and {record_id} segments in this sequence,
-    but these are not required.  Any segments or query parameters beyond the specified
+    but these are not required. Any segments or query parameters beyond the specified
```

Comment:
Was this change intentional?  If not, undo.

Reply:
Was the double spaces intentional? I removed it because that was odd to me